src/System.Data.Extensions/tests/SmokeTest.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Data
+{
+    public class SmokeTests
+    {
+        [Fact]
+        public void SmokeTest()
```

Comment:
Yes, @JeremyKuhne is looking for the real tests. SmokeTest is a placeholder for now to prove that we are able to add in test functionality